ee/docs/Backup-And-Recovery/backup_and_recovery_ceph_block_storage_rbd_mirroring.md

```diff
@@ -0,0 +1,191 @@
+# Ceph Block Storage Backup And Recovery
```

Comment:
Let's remove the `backup_and_recovery_` prefix from the files, they are already in a folder named like that.

Reply:
+1